### DISCUSSION TITLE
Welcome to ntop Discussions!

### DISCUSSION THREAD

**Discussion by syswe** [devopswe/ntop#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tunahancelik** [comment]
Hello ntop Community!

As a frequent user of ntop for monitoring Kubernetes nodes and pods, I've thought about potential UI/UX enhancements that could benefit us all. I'd like to share some ideas and invite community feedback and suggestions:

Customizable Dashboards: Allowing users to personalize the display of data and widgets to better suit individual needs.

Color-Blind Friendly Design: Adjusting the color scheme for better accessibility, aiding those with color vision deficiencies.

Mobile Compatibility: Improving the interface for mobile devices to ensure easy access from any device.

These are just starting points, and I'm eager to hear your thoughts and additional suggestions. Let's collaborate to make ntop even more user-friendly and accessible!

Thank you!
